src/Tools/dotnet-user-secrets/src/Internal/InitCommand.cs

```diff
@@ -120,19 +120,18 @@ public void Execute(CommandContext context)
                 }
 
                 // Add UserSecretsId element
+                propertyGroup.Add("  ");
                 propertyGroup.Add(new XElement("UserSecretsId", newSecretsId));
+                propertyGroup.Add($"{Environment.NewLine}  ");
```

Comment:
Same here, do we need to explicitly add a newline?

Reply:
Without it, we end up with the following after adding the `UserSecretsId` element:

```xml
<Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFrameworks>netcoreapp5.0</TargetFrameworks>
    
    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
    <UserSecretsId>595958db-f6e9-463c-879e-e03615a37179</UserSecretsId></PropertyGroup>

  <ItemGroup>
    <Compile Include="**\*.cs" Exclude="Excluded.cs;$(DefaultItemExcludes)" />
  </ItemGroup>
</Project>
```

It's basically needed because whitespace is preserved when loading the XML doc now.

Reply:
Ah, ok. My main concern is just that we don't actually know the incoming indentation level. If the user was indenting with 4 spaces, then we'll still end up messing that up. I don't really want us to add a whole complex new indentation engine here though :). This seems like a reasonable enough compromise to me now.

Reply:
I agree 100%. I didn't want to go overboard with trying to determine the appropriate number of spaces.